c/meterpreter/source/server/win/metsrv_test.c

```diff
@@ -76,5 +76,10 @@ int main(int argc, char **argv)
 
 	} while (0);
 
+	if (dllPath)

```

Comment:
no need for the check here, just freeing is fine

Reply:
Fixed